lib/bulbs-elements/util/dimensions.js

```diff
@@ -38,6 +38,26 @@ export function getScrollOffset () {
   return { x, y };
 }
 
+export function getRoundedBoundingClientRect (element) {
+  // firefox has too much precision and will create unexpected behavior when
+  //  comparing bounding rect pixel values. ensure the math always works out the
+  //  same by rounding. who's rendering 10000ths of a pixel anyways?
+  let rect = element.getBoundingClientRect();
+
+  // NOTE: cannot copy ClientRect in a normal way because it's a special object
+  return [
```

Comment:
A wonky way of doing this, but `ClientRect` cannot be copied in the normal ways because it's a special object. Let me know if you have a cleaner way of doing this.